deconz/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN apt-get update \
         libqt5websockets5 \
         libqt5widgets5 \
         sqlite3 \
+	iputils-ping \
+	iproute2 \
```

Comment:
why iproute2 ?

Reply:
because the command `ip` is in that package, and deconz is using that for the detecting of working internet